### PR TITLE
Merge with develop branch and bump the version

### PR DESCRIPTION
**Pull request by b00f**
[CHANGELOG](https://github.com/gallactic/gallactic/blob/7ac7972079075c8053c8ba0f061a1879ab2bc74a/CHANGELOG.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+
 You need to make sure you have install [Go](https://golang.org/) (version 1.10.1 or higher) and [rust](https://www.rust-lang.org). After installing them, you can follow these steps to compile and build the gallactic project:
 
-```
+```bash
 mkdir -p $GOPATH/src/github.com/gallactic/gallactic
-cd $GOPATH/src/github.com/gallactic
+cd $GOPATH/src/github.com/gallactic/gallactic
 git clone https://github.com/gallactic/gallactic.git .
 make
```
Comment (ToniyaSundaram) [inline README.md]: after git clone 
`cd gallactic` should be included else it cannot find the make file .

Comment (b00f) [reply README.md]: `cd $GOPATH/src/github.com/gallactic/gallactic` in this approach we are in the `galalctic/galalctic` folder. we clone the project in same folder. Look at the dot at the end of clone command. It means clone here.

File: cmd/gallactic/init.go [hunk 2]
```diff
@@ -71,12 +72,14 @@ func makeGenesis(workingDir string, chainName string) *proposal.Genesis {
 	for i := 0; i < len(accs); i++ {
 		k := key.GenAccountKey()
 		acc, _ := account.NewAccount(k.Address())
+		acc.AddToBalance(10000000000000000000)
+		acc.SetPermissions(permission.AllPermissions)
```
Comment (b00f) [inline cmd/gallactic/init.go]: @ToniyaSundaram : setPermissions here

File: README.md (hunk 1 above)
Comment (ToniyaSundaram) [reply README.md]: okay great thanks

**Pull request merged**